example_scripts/internal_tide_pycnocline_diagnostics.py

```diff
@@ -26,6 +26,8 @@
     config = 'AMM60'
     dir_AMM60 = "/projectsa/COAsT/NEMO_example_data/AMM60/"
     fil_nam_AMM60 = "AMM60_1d_20100704_20100708_grid_T.nc"
+    config_t = "/work/jelt/GitHub/COAsT/example_files/example_t_nemo_config.json"    
```

Comment:
Before this gets merged, can we change this to something generic (i.e. using "~") and platform-independent (i.e. using Path)?